### DISCUSSION TITLE
RDF prefix not exported in .ttl

### DISCUSSION THREAD

**Discussion by vtoure** [RDFLib/rdflib#1606] [Q&A]
Hi all, 

I noticed that RDFlib version 5.0.0 does not export the RDF prefix in the Turtle file (even when explicitly coded with `g.bind("rdf", RDF)`). It worked with v4.2.2. It is a bug or done on purpose?

**Comment by tgbugs** [comment]
In 5.0.0 unless a prefix is used in the file (`a` does not count) then it will not be round tripped. You can specify which/all prefixes to roundtrip via the rountrip_prefixes attribute https://github.com/RDFLib/rdflib/blob/c5ff12717c02cf21abc8dd00ed8f0ec27071ba5d/rdflib/plugins/serializers/turtle.py#L45. Here is how it works https://github.com/RDFLib/rdflib/blob/c5ff12717c02cf21abc8dd00ed8f0ec27071ba5d/rdflib/plugins/serializers/turtle.py#L118-L125.
